tests/bears/BearTest.py

```diff
@@ -129,6 +131,24 @@ def test_bad_bear(self):
         # debug message contains custom content, dont test this here
         self.queue.get()
 
+    def test_LocalBear(self):  # Fails because of no run() implementation
+        self.uut = LocalBear(self.settings, self.queue)
+        self.uut.execute('filename', 'file\n')
+        self.check_message(LOG_LEVEL.DEBUG)
+        self.check_message(LOG_LEVEL.WARNING,
+                           'Bear LocalBear failed to run on file filename. '
+                           'Take a look at debug messages (`-V`) for further '
+                           'information.')
+
+    def test_GlobalBear(self):  # Fails because of no run() implementation
```

Comment:
Because in that case it gets into except block, then again the except `NotImplementedError` block, and the if else will never be executed ?